Model/src/main/java/org/gluu/oxauth/model/authorize/AuthorizeRequestParam.java

```diff
@@ -38,6 +38,7 @@
     String CODE_CHALLENGE_METHOD = "code_challenge_method";
     String CUSTOM_RESPONSE_HEADERS = "custom_response_headers";
     String AUTH_REQ_ID = "auth_req_id";
+    String USER_CODE = "user_code";
```

Comment:
We shouldn't have it anymore here. It's not part of request. We should have internal SESSION_USER_CODE.